src/proposal/timeline.rs

```diff
@@ -30,48 +125,49 @@ impl TimelineStatus {
     }
 
     pub fn is_review(&self) -> bool {
-        matches!(self, TimelineStatus::Review(..))
+        matches!(self, TimelineStatusV1::Review(..))
```

Comment:
I would consider having a type-alias:

```rust
type TimelineStatus = TimelineStatusV2;
```

This way with the future changes to the timeline, we won't need to change all the references to TimelineStatusV1 to TimelineStatusV2